src/main/java/io/github/techstreet/dfscript/script/action/ScriptActionType.java

```diff
@@ -119,6 +120,31 @@ public enum ScriptActionType {
             ctx.value("Value")
         ))),
 
+    GET_REQUEST(builder -> builder.name("Get Webrequest")
+        .description("Makes a get request to the internet.")
+        .icon(Items.GRASS_BLOCK)
+        .arg("Result", ScriptActionArgumentType.VARIABLE)
+        .arg("Url", ScriptActionArgumentType.TEXT)
+        .category(ScriptActionCategory.VARIABLES)
+        .action(ctx -> {
+            try{
+            StringBuilder result = new StringBuilder();
+            URL url = new URL(ctx.value("Url").asText());
+            HttpURLConnection conn = (HttpURLConnection) url.openConnection();
+            conn.setRequestMethod("GET");
+            try (BufferedReader reader = new BufferedReader(new InputStreamReader(conn.getInputStream()))) {
+              for (String line; (line = reader.readLine()) != null; ) {
+                result.append(line);
+              }
+            }
+            ctx.task().context().setVariable(ctx.variable("Result").name(), new ScriptTextValue(result.toString()));
+          }catch(MalformedURLException ex){
+
```

Comment:
A malformed URL should create an error in something like `ChatUtil.error` or preferably `OverlayManager.getInstance().add`